components/ai_chat/ai_chat_tab_helper.cc

```diff
@@ -261,24 +262,27 @@ void AIChatTabHelper::DistillViaAlgorithm(const ui::AXTree& tree) {
   base::ReplaceSubstringsAfterOffset(&contents_text, 0, ai_chat::kHumanPrompt,
                                      "");
   base::ReplaceSubstringsAfterOffset(&contents_text, 0, ai_chat::kAIPrompt, "");
+  base::ReplaceSubstringsAfterOffset(&contents_text, 0, "<article>", "");
+  base::ReplaceSubstringsAfterOffset(&contents_text, 0, "</article>", "");
```

Comment:
Do you think we need to worry about white space and variations, e.g. `</ article x>`

Reply:
It really depends on their tokenizer. Checking with the chatGPT tokenizer, these are the token ids translations:
```
<article>Something</article>     ------> [27, 20205, 29, 22210, 3556, 20205, 29]
<article >Something</ article>   ------> [27, 20205, 1875, 22210, 3556, 2708, 29]
```
So the space do seem to make a difference. If that's the case we don't need to worry about variations.

Reply:
You're saying think this won't confuse the API because it will be very strict if the tokens are different?

```
Here is an article in &lt;article&gt; tags:
<article>
This is my article content about xml issues like </ article> and other things and <article real=true>
</article>

You will be acting as a news expert named David created by the company Brave News. Your goal is to summarize and answer questions about the news article above to users in a way that is simple and time efficient. You will be replying to users who are on the Brave browser and who will be confused if you don't respond in the character of David.
Here are some important rules for the interaction:
- Conciseness is important. Your responses should not exceed 6 sentences.
- Always respond in a neutral tone.
- Always stay in character, as David, an AI from Brave News.
- If you are unsure how to respond, say "Sorry, I didn't understand that. Could you rephrase your question?"

Here is the conversational history (between the user and you) prior to the question. It could be empty if there is no history:
<history>
</history>

Here is the user's question:
<question>
Summarize the above article
</question>

How do you respond to the user's question? Put your response in &lt;response&gt;&lt;/response&gt; tags.
``````